### DISCUSSION TITLE
nuclei skips all targets

### DISCUSSION THREAD

**Discussion by catchmeifyoucaan** [projectdiscovery/nuclei#3132] [Q&A] [answered]
hello, I was using the nuclei tool yesterday and i found the new update on twitter, immediately i updated, nuclei started ignoring all targets in a folder, specifically if i try to add -s info, it will work on few targets but when i use other methds such as -s medium,high,critical it automatically keeps skipping them 
![nuclei](https://user-images.githubusercontent.com/94643805/210297231-ddb7bcec-44b8-4be7-9bab-3577cb18651e.png)

**Comment by ehsandeep** [comment, marked as answer]
@catchmeifyoucaan this is not a bug and is the result of `-mhe` feature; what actually changed in the last release is displaying this information on default run instead of verbose mode, so now you can see there is something wrong with hosts where they reject the incoming request from nuclei, either because of speed or something else that you can dig further by slowing down your scan speed with `rate-limit` options or you can share the list with us to review it further.

```console
nuclei -h mhe

Usage:
  nuclei [flags]

Flags:
   -mhe, -max-host-error int  max errors for a host before skipping from scan (default 30)
```

**Reply by jaikishantulswani** [reply]
@ehsandeep Can it be possible to force scan all hosts or any flag to skip-check for `mhe`

**Reply by ehsandeep** [reply]
https://github.com/projectdiscovery/nuclei/issues/3138

**Comment by catchmeifyoucaan** [comment]
thank you, i got it now

**Comment by catchmeifyoucaan** [comment]
the oprion -no-mhe is not available yet, till next update?

nuclei -l priv -s critical -no-mhe
flag provided but not defined: -no-mhe

**Comment by amanmahendra00** [comment]
Hi @ehsandeep It is skipping all valid hosts why?

**Reply by Universal-God-One** [reply]
same  case

**Reply by GeorginaReeder** [reply]
Hi @Universal-God-One ! Could you give us a bit more detail on what you're having trouble with, including what you were running in nuclei exactly that caused all valid hosts to skip?

Better yet, if you're able to create a new issue for this, including screenshots, that would be really helpful. Thanks!

**Reply by Universal-God-One** [reply]
i am  using nuclei  version3.2.5
even for  normal  search  also   and  it  also cause  my internet connection lost....... and  i am  not  ableto  search  internet in kali.......

![nuclei_error](https://github.com/projectdiscovery/nuclei/assets/168637168/c09f3e4c-ae9f-4605-a197-6a48a704a09e)

**Reply by GeorginaReeder** [reply]
Thanks for that @Universal-God-One!

@ehsandeep , since this is an older issue, I'm not sure what might be causing the trouble here?

**Comment by ehsandeep** [comment]
@catchmeifyoucaan @amanmahendra00 this is now fixed with https://github.com/projectdiscovery/nuclei/releases/tag/v2.8.5

**Comment by catchmeifyoucaan** [comment]
Well I don't think so, the -no-mhe was not added and its still skipping almost all targets
![Uploading 20230103_233732.jpg…]()

**Comment by catchmeifyoucaan** [comment]
![20230103_233732](https://user-images.githubusercontent.com/94643805/210438435-fb859b67-5b3f-42da-92bf-e71407abdaf5.jpg)
